README.md

```diff
@@ -46,43 +43,40 @@ Contributions:
 ## Acknowledgements:
 
     1. https://simpleisbetterthancomplex.com/tutorial/2016/08/01/how-to-upload-files-with-django.html
-    2. https://stackoverflow.com/questions/3003146/best-way-to-write-an-image-to-a-django-httpresponse 
-    3. Photo use: https://thenounproject.com/term/question/1101884/ Author: AliWijaya. This file is licensed under the 	Creative Commons Attribution-Share Alike 3.0 Unported license. 
+    2. https://stackoverflow.com/questions/3003146/best-way-to-write-an-image-to-a-django-httpresponse
+    3. Photo use: https://thenounproject.com/term/question/1101884/ Author: AliWijaya. This file is licensed under the 	Creative Commons Attribution-Share Alike 3.0 Unported license.
     4. All other photos are licensed under free use.
     5. https://stackoverflow.com/questions/29321494/show-input-field-only-if-a-specific-option-is-selected/29321711. Author: https://stackoverflow.com/users/4721273/josephus87
     6. https://www.w3schools.com/howto/howto_css_fixed_sidebar.asp
     7. https://stackoverflow.com/questions/15950007/centering-floated-images-in-div. author: https://stackoverflow.com/users/2157321/dhaval-marthak*/
-    
 
 # Squawk Documentation
 
-Squawk is a distributed social network that gives control back to the user. With Squawk, you get all the features you expect from a modern social network without sacrificing privacy or being flooded with promotional content. For every post you create, you can choose exactly who can view it. When viewing content, you can access the Explore page to see all public posts connected to your node, or head to Your Feed to see only the currated content that you choose to follow. 
+Squawk is a distributed social network that gives control back to the user. With Squawk, you get all the features you expect from a modern social network without sacrificing privacy or being flooded with promotional content. For every post you create, you can choose exactly who can view it. When viewing content, you can access the Explore page to see all public posts connected to your node, or head to Your Feed to see only the currated content that you choose to follow.
 
 ## User Access
-The website is hosted by Heroku at https://cmput-404.herokuapp.com/ 
 
-Alternatively, one can run the app locally by running the following command from the root directory:
- 	
-	virtualenv venv --python=python3
-	source venv/bin/activate
-	pip3 install -r requirements.txt
-	python3 social_distribution/manage.py runserver
+The website is hosted by Heroku at https://cmput-404.herokuapp.com/
 
+Alternatively, one can run the app locally by running the following command from the root directory:
+virtualenv venv --python=python3
+source venv/bin/activate
+pip3 install -r requirements.txt
+python3 social_distribution/manage.py runserver
 
```

Comment:
this is no longer accurate. `manage.py` is not longer in a subfolder. Also, perhaps we should say `heroku local`